infrastructure/transactional-emails/src/main.ts

```diff
@@ -55,39 +58,17 @@ class TransactionalEmails extends TerraformStack {
       tags: config.tags,
     });
 
-    const userEventTopicArn = `arn:aws:sns:${region.name}:${caller.accountId}:${config.eventBridge.prefix}-${config.environment}-${config.eventBridge.userTopic}`;
-    this.subscribeSqsToSnsTopic(
-      sqsLambda,
-      snsTopicDlq,
-      userEventTopicArn,
-      config.eventBridge.userTopic,
-    );
-
-    const premiumPurchaseTopicArn = `arn:aws:sns:${region.name}:${caller.accountId}:${config.eventBridge.prefix}-${config.environment}-${config.eventBridge.premiumPurchaseTopic}`;
-    this.subscribeSqsToSnsTopic(
-      sqsLambda,
-      snsTopicDlq,
-      premiumPurchaseTopicArn,
-      config.eventBridge.premiumPurchaseTopic,
-    );
-
-    const userRegistrationTopicArn = `arn:aws:sns:${region.name}:${caller.accountId}:${config.eventBridge.prefix}-${config.environment}-${config.eventBridge.userRegistrationTopic}`;
-    this.subscribeSqsToSnsTopic(
-      sqsLambda,
-      snsTopicDlq,
-      userRegistrationTopicArn,
-      config.eventBridge.userRegistrationTopic,
-    );
+    const topicArns = [];
```

Comment:
I have a bias for not mutating when possible, but this is a super small nit. you can use `map` instead of `for...push`

Reply:
oh. yea of course. idk why i didnt map..